static/js/components/SourceDesktop.jsx

```diff
@@ -110,6 +110,7 @@ export const useSourceStyles = makeStyles((theme) => ({
   },
   followupContainer: {
     display: "flex",
+    overflowX: "scroll",
```

Comment:
You should apply the same change to the style rules in SourceMobile.jsx

Reply:
@dannygoldstein was this you?

Reply:
Good OK added this to mobile.